### PR TITLE
 [Qt6] Update to 6.1.1

### PR DESCRIPTION
**Pull request by Neumann-A**
Running CI test for it. 
- [x] depends #18377

### CODE REVIEW

**Comment by JackBoosY** [comment]
Please ping me if this PR is ready for review.

**Comment by Neumann-A** [comment]
@JackBoosY: You can review it. I don't plan to add any new changes. It just not marked ready for review due to also containing #18377

**Review by JackBoosY — APPROVED** [review]

File: ports/qtbase/cmake/qt_port_details.cmake [hunk 1]
```diff
@@ -30,45 +30,46 @@ set(QT_PORTS qtbase
...
+set(qtscxml_REF            666adc604fec06fa5b38be4d4f1b0e9b56f16c2b)
+set(qtvirtualkeyboard_REF  5c7df0c55a96e9855bd27c23a18d6f2d91305d31)
+set(qtcharts_REF           251d18d960a6f6ee04f07ba338beb317c425e0a2)
 
 if(QT_UPDATE_VERSION)
     message(STATUS "Running Qt in automatic version port update mode!")
     set(_VCPKG_INTERNAL_NO_HASH_CHECK 1)
     if("${PORT}" MATCHES "qtbase")
-        foreach(_current_qt_port IN LISTS QT_PORTS)
+        foreach(_current_qt_port IN LISTS QT_PORTS )
```
[the remark this answers is not in the document]
Comment (Neumann-A) [reply ports/qtbase/cmake/qt_port_details.cmake]: No. I though about adding qt here but added it into QT_PORTS

**Review by JackBoosY — APPROVED** [review]

**Comment by BillyONeal** [comment]
Thanks for your contribution!

**Pull request merged**